frontend/main/.eslintrc

```diff
@@ -1,7 +1,40 @@
 {
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
   "extends": [
-    "react-important-stuff",
+    "eslint:recommended",
+    "plugin:react/recommended",
```

Comment:
I think you should add this to the extension list `"airbnb/hooks"`. Airbnb have some rules on how to use react hooks.

Reply:
I'll do that